### PR TITLE
Fix invite display

### PR DESCRIPTION
**Pull request by KN4CK3R**
Regression of #23901

Before
grafik

After
grafik

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Comment by wxiaoguang** [comment]
Which page? It's related to "float-right". I think we should fix it as "right"

File: templates/org/team/members.tmpl [hunk 1]
```diff
@@ -49,12 +49,12 @@
...
 					{{range .Invites}}
-						<div class="item">
-							{{.Email}}
+						<div class="item gt-df gt-ac gt-fw">
+							<span class="gt-f1">{{.Email}}</span>
 							<form action="{{$.OrgLink}}/teams/{{$.Team.LowerName | PathEscape}}/action/remove_invite" method="post">
 								{{$.CsrfTokenHtml}}
 								<input type="hidden" name="iid" value="{{.ID}}">
-								<button class="ui red button gt-float-right">{{$.locale.Tr "org.members.remove"}}</button>
+								<button class="ui red button delete-button">{{$.locale.Tr "org.members.remove"}}</button>
```
Comment (wxiaoguang) [inline templates/org/team/members.tmpl]: `delete-button` seems not right. it doesn't work like this.

Comment (KN4CK3R) [reply templates/org/team/members.tmpl]: Then most/all(?) buttons with `red button` and `delete-button` are wrong?

Comment (wxiaoguang) [reply templates/org/team/members.tmpl]: There is JS code for `delete-button`, it works another way.

If you add `delete-button` to a form button, the form won't be submitted.

Comment (silverwind) [reply templates/org/team/members.tmpl:57]: Yes, gotta be careful with such generic class names.

**Review by lafriks — APPROVED** [review]

**Pull request merged**